### ISSUE TITLE
[process id]: [issue summary] How to create a raster cube with given extent/resolution?

### ISSUE THREAD

**Issue by pierocampa** [Open-EO/openeo-processes#381]
**Process ID:** `create_raster_cube` / `add_dimension`

**Describe the issue:**
Is there a way to define a cube with a given spatio-temporal extent and resolution?
This way I could use it as target cube of some other process (e.g. `resample_cube_spatial` ?)

I also do not understand the possible applications of the `create_raster_cube` process in its current form. !
Thanks for any help!

**Additional context**
Coming from the OGC-WCPS world, I am thinking about the would-be equivalent [coverage constructor expression](https://docs.ogc.org/is/08-068r3/08-068r3.html#66).

**Comment by m-mohr** [comment]
No, not easily or not at all, I think. If you want to resample to a given resolution, use resample_spatial. To limit the extent use filter_spatial or filter_bbox.
